src/App.tsx

```diff
@@ -115,6 +116,10 @@ class App extends React.Component<AppProps> {
           <this.PrivateRoute path={ROUTES.REPORT} component={Report} />
           <this.PrivateRoute path={ROUTES.LOGS} component={Logs} />
           <this.PrivateRoute path={ROUTES.RACK_PRINT} component={RackView} />
+          <this.PrivateRoute
+            path={ROUTES.CHANGE_PLAN}
```

Comment:
not sure if this is expected, but I get "The page that you are looking for does not exist!" message on detail page

Reply:
yes good catch i accidentally had an unpushed commit im SORRY

Reply:
it be like that somtimes